src/system/SecurityCenterModule/Helper/PurifierHelper.php

```diff
@@ -51,14 +51,9 @@ public function getPurifierConfig($args)
         $config = $this->getPurifierDefaultConfig();
         if (!isset($args['forcedefault']) || true !== $args['forcedefault']) {
             $savedConfig = $this->variableApi->get('ZikulaSecurityCenterModule', 'htmlpurifierConfig');
-
             if (!is_null($savedConfig) && false !== $savedConfig) {
-                $savedConfig = unserialize($savedConfig);
-                foreach ($savedConfig as $section => $values) {
-                    foreach ($values as $k => $v) {
-                        $config->set($section . '.' . $k, $v);
-                    }
-                }
+                /** @var \HTMLPurifier_Config $config */
+                $config = unserialize($savedConfig);
```

Comment:
because the old method wasn't working - it couldn't iterate over the object (`as $section => $values)`) So it failed. I'm open to better ideas.